launcher_scripts/conf/training/gpt3/2b.yaml

```diff
@@ -0,0 +1,210 @@
+run:
+  name: gpt3_2b
+  results_dir: ${base_results_dir}/${.name}
+  time_limit: "1-00:00:00"
+  dependency: "singleton"
+
+trainer:
+  num_nodes: 8
+  devices: 8
+  accelerator: gpu
+  precision: bf16
+  logger: False # logger provided by exp_manager
+  enable_checkpointing: False
+  replace_sampler_ddp: False
+  max_epochs: null
+  max_steps: 300000 # consumed_samples = global_step * global_batch_size
```

Comment:
We may need to re-compute max steps to account for 2048 seq length and a smaller GBS.